Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.42.0-buster as builder
+FROM rust:1.44.0-buster as builder
```

Comment:
👍

Reply:
Yeah, I think we need to add a reminder to our calendars to update the docker file one week after a new rust version rolls out.